mmedit/models/backbones/encoder_decoders/decoders/aot_decoder.py

```diff
@@ -0,0 +1,49 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class AOTDecoder(nn.Module):
+    """Decoder used in AOT-GAN model.
+    This implementation follows:
+    Aggregated Contextual Transformations for High-Resolution Image Inpainting
+    Args:
+        in_channels (int): Channel number of input feature.
+        act_cfg (dict): Config dict for activation layer, "relu" by default.
+    """
+
+    def __init__(self, in_channels=256, act_cfg=dict(type='ReLU')):
+        super().__init__()
+
+        self.dec1 = ConvModule(
+            in_channels,
+            128,
+            kernel_size=3,
+            stride=1,
+            padding=1,
+            act_cfg=act_cfg)
+        self.dec2 = ConvModule(
+            128, 64, kernel_size=3, stride=1, padding=1, act_cfg=act_cfg)
```

Comment:
if `mid_channels` is 128 by default, maybe 64 can be changed to `mid_channels // 2`? I do not know much about the details of the network, just for your consideration.